managed/src/main/resources/v1.routes

```diff
@@ -148,7 +148,8 @@ POST    /customers/:cUUID/universes/:uniUUID/upgrade                           c
 POST    /customers/:cUUID/universes/:uniUUID/disk_update                       com.yugabyte.yw.controllers.UniverseController.updateDiskSize(cUUID: java.util.UUID, uniUUID: java.util.UUID)
 GET     /customers/:cUUID/universes/:uniUUID/status                            com.yugabyte.yw.controllers.UniverseController.status(cUUID: java.util.UUID, uniUUID: java.util.UUID)
 GET     /customers/:cUUID/universes/:uniUUID/health_check                      com.yugabyte.yw.controllers.UniverseController.healthCheck(cUUID: java.util.UUID, uniUUID: java.util.UUID)
-GET     /customers/:cUUID/universes/find/:universeName                         com.yugabyte.yw.controllers.UniverseController.findByName(cUUID: java.util.UUID, universeName: String)
+GET     /customers/:cUUID/universes/find/:universeName                         com.yugabyte.yw.controllers.UniverseController.findByName_Corrected(cUUID: java.util.UUID, universeName: String)
+GET     /customers/:cUUID/universes/checkUnique/:universeName                  com.yugabyte.yw.controllers.UniverseController.verifyUniqueName(cUUID: java.util.UUID, universeName: String)
```

Comment:
Do we use camelCase in rest endpoints? talking about **checkUnique** not universeName, variables are fine.